src/lib/features/feature-toggle/tests/feature-toggle-strategies-store.e2e.test.ts

```diff
@@ -246,6 +247,9 @@ describe('strategy parameters default to sane defaults', () => {
         });
         expect(strategy.parameters.stickiness).toBe(defaultStickiness);
     });
+});
+
+describe('max metrics collection', () => {
     test('Read feature with max number of strategies', async () => {
```

Comment:
This test was previously in the describe block called "strategy parameters default to sane defaults". This seems like a better fit.